### PR TITLE
[jaeger-operator] Updated jaeger-operator helm chart to include priorityclass

### PR DESCRIPTION
**Pull request by sonofgibs**
#### What this PR does
Updates helm chart to include priorityclass for allinone as added here: https://github.com/jaegertracing/jaeger-operator/pull/2218 

#### Which issue this PR fixes

*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format,
will close that issue when PR gets merged)*

- fixes #

#### Checklist

- [x] [DCO](https://github.com/jaegertracing/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Commits are [GPG signed](https://docs.github.com/en/github/authenticating-to-github/about-commit-signature-verification)
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (`[jaeger]` or `[jaeger-operator]`)
- [x] README.md has been updated to match version/contain new values

### CODE REVIEW

File: charts/jaeger-operator/crds/crd.yaml [hunk 1]
```diff
@@ -14573,3 +14866,669 @@ spec:
...
+    - jaegertracing.io
+    apiVersions:
+    - v1
+    operations:
+    - CREATE
+    - UPDATE
+    resources:
+    - jaegers
+  sideEffects: None
+
```
Comment (gracedo) [inline charts/jaeger-operator/crds/crd.yaml:15534]: we shouldn't need all these manifests here -- just the CRD from https://github.com/jaegertracing/jaeger-operator/blob/main/bundle/manifests/jaegertracing.io_jaegers.yaml

File: charts/jaeger-operator/crds/crd.yaml [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 metadata:
-  name: jaegers.jaegertracing.io
+  annotations:
+    cert-manager.io/inject-ca-from: observability/jaeger-operator-serving-cert
+    controller-gen.kubebuilder.io/version: v0.9.2
   labels:
-    app.kubernetes.io/name: jaeger-operator
-    app.kubernetes.io/instance: jaeger-operator
+    name: jaeger-operator
+  name: jaegers.jaegertracing.io
```
Comment (gracedo) [inline charts/jaeger-operator/crds/crd.yaml:9]: need to fix formatting here

File: charts/jaeger-operator/Chart.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: jaeger-operator Helm chart for Kubernetes
 name: jaeger-operator
-version: 2.46.0
+version: 2.46.2
```
Comment (gracedo) [inline charts/jaeger-operator/Chart.yaml:4]: should this be 2.46.1?

Comment (sonofgibs) [reply charts/jaeger-operator/Chart.yaml]: My fork wasn't up to date: https://github.com/jaegertracing/helm-charts/blob/main/charts/jaeger-operator/Chart.yaml#L4

**Review by gracedo — APPROVED** [review]

**Comment by gracedo** [comment]
@batazor @cpanato would you be able to take a look when convenient please? The jaeger-operator CRD was out of date, so just updating it here. Thank you!

**Review by batazor — APPROVED** [review]

**Pull request merged**